pyQuARC/code/custom_validator.py

```diff
@@ -15,15 +15,15 @@ def ends_at_present_flag_logic_check(
         ends_at_present_flag, ending_date_time, collection_state
     ):
         collection_state = collection_state.upper()
-        if ends_at_present_flag == None:
+        if not ends_at_present_flag:
```

Comment:
this is incorrect. this will execute if the `ends_at_present_flag` is `False` too, which isn't what we want